README.md

```diff
@@ -72,6 +75,13 @@ Additionally, the following variables may be useful to override:
 - `geoip_local_path` - The directory to extract the database into.
 - `geoip_local_file` - The filename used to store the downloaded database.
 
+<!--
+| Variable | Description | Default | Required |
+|----------|-------------|---------|----------|
+| optional_variable | Describe its purpose. | `default_value` | No |
+| required_variable | Describe its purpose. | n/a | Yes |
+-->
+
```

Comment:
The role variable descriptions should be reworked into tabular format.

Reply:
Please enjoy aa8a916 with these changes.